datasets/MoistureMaterials.idf

```diff
@@ -6,85 +6,125 @@
 ! The objects and syntax in this dataset comply with EnergyPlus, Version 8.4.0 and above, unless otherwise noted.
 
   MaterialProperty:MoisturePenetrationDepth:Settings,
-    Gypsum,                  !- Name
-    0.004,                   !- Moisture Penetration Depth {m}
-    0.072549,                !- Moisture Equation Coefficient a {dimensionless}
-    0.397173,                !- Moisture Equation Coefficient b {dimensionless}
-    0.007774,                !- Moisture Equation Coefficient c {dimensionless}
-    11.7057;                 !- Moisture Equation Coefficient d {dimensionless}
-
+    Gypsum,                   !- Name
+    27.3345673099,            !- Water Vapor Diffusion Resistance Factor {dimensionless}
+    0.072549,                 !- Moisture Equation Coefficient a {dimensionless}
+    0.397173,                 !- Moisture Equation Coefficient b {dimensionless}
+    0.007774,                 !- Moisture Equation Coefficient c {dimensionless}
+    11.7057,                  !- Moisture Equation Coefficient d {dimensionless}
+    0.004,                    !- Surface-layer penetration depth {m}
+    0,                        !- Deep-layer penetration depth {m}
+    0,                        !- Coating layer thickness {m}
+    0;                        !- Coating layer water vapor diffusion resistance factor {m}
+    
```

Comment:
@jdwoods21, I just found this file that needed to be updated. It's supposed to contain EMPD input data for a collection of materials. I went ahead and updated it using our transition script and some material density data from `datasets/ASHRAE_2005_HOF_Materials.idf`, Wolfram Alpha (vinyl), and Wikipedia (soil). I don't know if you need to update these with appropriate deep layer stuff.